src/olympia/reviews/views.py

```diff
@@ -65,12 +66,16 @@ def review_list(request, addon, review_id=None, user_id=None, template=None):
     else:
         ctx['page'] = 'list'
         qs = qs.filter(is_latest=True)
+        # Don't filter out empty reviews for admins.
+        if not is_admin:
+            # But otherwise, filter out everyone elses empty reviews.
+            qs = qs.filter(~Q(body=None) | Q(user=request.user.pk))
```

Comment:
It doesn't matter if it's an `AnonymousUser` - there can't be any reviews (because user shouldn't be none, as you said) and worst case we're just not hiding them, as now.  Seems an unnecessary extra check, but meh, can do.